<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui</class>
 <widget class="QWidget" name="ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="only_rbt">
       <property name="text">
        <string>only_rbt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="multi_rbt">
       <property name="text">
        <string>multi_rbt</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_list1" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_list1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>add_list1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_list1"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="Constraint_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="reset_pushButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_parentConstraint">
          <property name="text">
           <string>parentConstraint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_pointConstraint">
          <property name="text">
           <string>pointConstraint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_orientConstraint">
          <property name="text">
           <string>orientConstraint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_scaleConstraint">
          <property name="text">
           <string>scaleConstraint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_AimConstraint">
          <property name="text">
           <string>AimConstraint</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_list2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_list2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>add_list2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_list2"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_parentContraint">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>ParentContraint</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
